src/lib.rs

```diff
@@ -125,9 +122,19 @@ mod rectangle;
 mod region;
 mod surface;
 mod matrices;
+mod support;
```

Comment:
Seems this better have `#[cfg(any(feature = "pdf", feature = "svg", feature = "ps", feature = "dox"))]`

Reply:
Good point, commit incoming.